.gitignore

```diff
@@ -104,5 +104,16 @@ wheelhouse/
 .pytest_cache
 
 # by default, ignore in-toto metadata from the command line
-.links/
+.in-toto/
 *.link
+
+# and most TUF metadata and all targets
+# https://www.atlassian.com/git/tutorials/saving-changes/gitignore#git-ignore-patterns
+datadog_checks_downloader/datadog_checks/downloader/data/repo/metadata/current/*.json
+!datadog_checks_downloader/datadog_checks/downloader/data/repo/metadata/current/.gitignore
+!datadog_checks_downloader/datadog_checks/downloader/data/repo/metadata/current/root.json
+datadog_checks_downloader/datadog_checks/downloader/data/repo/metadata/previous/*.json
+!datadog_checks_downloader/datadog_checks/downloader/data/repo/metadata/previous/.gitignore
+datadog_checks_downloader/datadog_checks/downloader/data/repo/targets/simple/
+datadog_checks_downloader/datadog_checks/downloader/data/repo/targets/in-toto-*/
+!datadog_checks_downloader/datadog_checks/downloader/data/repo/targets/.gitignore
```

Comment:
What's the purpose? I don't think these affect us currently

Reply:
Hopefully, this is answered in the comment above. It did affect @therve the other day, and prevented him from releasing datadog-checks-downloader...

Reply:
Where do they come from?